demisto_sdk/__main__.py

```diff
@@ -1378,6 +1378,8 @@ def init(**kwargs):
     "--old-version", help="Path of the old integration version yml file.")
 @click.option(
     "--skip-breaking-changes", is_flag=True, help="Skip generating of breaking changes section.")
+@click.option(
+    "--custom-image-link", help="A custom link to a playbook image. If not stated, a default link will be added to the file.")
```

Comment:
is it a link, or path? (a link takes one somewhere when clicked, while path shows where to go. I prefer the latter)

Reply:
it can be relative or non-clickable and takes to an image, please replace with `path` which is clearer.